config/default/image.style.agency_blog_landing_page.yml

```diff
@@ -8,14 +8,8 @@ effects:
   1e57e9b6-f804-4f4b-bf66-ca14b88e2c31:
     uuid: 1e57e9b6-f804-4f4b-bf66-ca14b88e2c31
     id: image_scale_and_crop
-    weight: -9
+    weight: 1
     data:
       width: 346
       height: 277
       anchor: center-center
-  dbbfc328-b847-48c2-a309-eceb279b43bb:
```

Comment:
Please verify whether we need these changes or not.

Reply:
According to this - yes https://github.com/ytgov/yukon-ca/pull/51/files#diff-fb4a3c3853b32cd27cfeead5a504fb45e2a75428a9e0ed71c3a881b8184a3c5a

Reply:
reverted